app/mailers/notify_mailer.rb

```diff
@@ -21,4 +21,15 @@ def new_api_key_request(user)
 
     mail(to: 'registerteam@digital.cabinet-office.gov.uk')
   end
+
+  def register_download_confirmation(download_user)
+    set_template('67a7d2f9-3faf-431a-a2a1-812f40a72e22')
+
+    set_personalisation(
+      register_name: download_user.register.capitalize,
+      register_download_link: "https://www.registers.service.gov.uk/registers/#{download_user.register.parameterize}/download-csv"
```

Comment:
frontend could send this link in its response to self-service, rather than having the resolver logic in two places, but I think this is fine for now.

Reply:
I actually done this work in the frontend and then moved it to selfservice because we also send the API key emails from selfservice...

Reply:
Yeah, I am thinking `registers-frontend` sends a param in the post e.g. `download_url` which registers-selfservice can use.

Reply:
Ah I see 👍

Reply:
I think this is OK for now and we can pick this up as a possible future TODO.